src/components/Profile.tsx

```diff
@@ -0,0 +1,39 @@
+import Navbar from '@/components/Navbar';
+import { AuthProps } from '@/types/types';
+import { PassageUser } from '@passageidentity/passage-elements/passage-user';
+import dotenv from 'dotenv';
+import Router from "next/router";
+import { useEffect } from 'react';
+
+function Profile({isAuthorized, userID}: AuthProps) {
+  useEffect(() => {
+    require("@passageidentity/passage-elements/passage-profile");
+    if (!isAuthorized) {
+      Router.push("/"); 
+    }
+  },[isAuthorized]);
+
+  const signOut = async () => {
+    if (userID !== 'ABrrCENR3M0I6XZ7NLA7gNCY'){
+      new PassageUser().signOut();
+    }
+    Router.push("/");
+  };
+
+  dotenv.config();
+  const passageAppId = process.env.NEXT_PUBLIC_PASSAGE_APP_ID;
+
+  if (!passageAppId) {
+    throw new Error('NEXT_PUBLIC_PASSAGE_APP_ID is not defined in the environment.');
+  }
+
+  return (
+    <div className='flex flex-col content-center'>
```

Comment:
On certain screen sizes I am not able to scroll down and get to the log out button because the nav bar at the bottom has a fixed position. I think if you just add a chunky margin like `mb-20` to the bottom of this div, then it should solve the issue.